databricks/koalas/internal.py

```diff
@@ -422,32 +422,32 @@ def attach_default_index(sdf):
         There are three types of default index that can be controlled by `DEFAULT_INDEX`
         environment variable.
 
-        - one-by-one: It implements an one-by-one sequence by Window function without
+        - sequence: It implements an sequence sequence by Window function without
```

Comment:
what do you mean by squence sequence?  you might want to just give some examples explaining the differences.

Reply:
Sorry, I will update it soon.